### PR TITLE
#2 Initial setup Dioxus

### PR DESCRIPTION
**Pull request by ShresthaRujal**

### CODE REVIEW

File: .github/workflows/rust.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 name: Rust
 
-on: [push, pull_request]
+on: [pull_request]
```
Comment (shakyaprasen) [inline .github/workflows/rust.yml:3]: any reason to remove push? Adding push will cause it to build on new commits pushed to PR as well

File: Readme.md [hunk 2]
```diff
@@ -3,5 +3,7 @@
 ## How to run
 1. Prerequisites:
    - Install rust using rustup
+   - To run the TailwindCSS in watch mode `npx tailwindcss -i ./src/style.css -o ./dist/output.css --watch`
```
Comment (shakyaprasen) [inline Readme.md:6]: do we have to run this command along with cargo watch/run too?

**Review by shakyaprasen — APPROVED** [review]

File: Readme.md (hunk 2 above)
Comment (ShresthaRujal) [reply Readme.md:6]: yes. it watches changes in css and compile into CSS

File: .github/workflows/rust.yml (hunk 1 above)
Comment (ShresthaRujal) [reply .github/workflows/rust.yml:3]: I do not think building twice the same thing is necessary for both PUSH and PULL_REQUEST.

**Pull request merged**